core/roboconf-target-iaas-occi/src/main/java/net/roboconf/target/occi/internal/OcciVMUtils.java

```diff
@@ -347,9 +382,12 @@ public static String createCloudAutomationVM(String hostIpPort, String image, St
 						+ "\"occi.compute.userdata\": \"" + userDataScript + "\",\n"
 						+ "\"occi.category.title\": \"scriptMixin\"\n"
 					+ "}\n}\n}";
+
+			final Logger logger = Logger.getLogger( OcciVMUtils.class.getName());
 			logger.finest(request);
-			httpURLConnection.setRequestProperty("Content-Length", "" +
-					Integer.toString(request.getBytes().length));
+			httpURLConnection.setRequestProperty(
```

Comment:
Sure ? It used to work, still the case ? Same advice for all "utf8"-related modifications...